src/TextForEvent.js

```diff
@@ -244,15 +244,16 @@ function textForPowerEvent(event) {
     }
     return _t('%(senderName)s changed the power level of %(powerLevelDiffText)s.', {
         senderName: senderName,
-        powerLevelDiffText: diff.join(", ")
+        powerLevelDiffText: diff.join(", "),
     });
 }
 
 function textForWidgetEvent(event) {
-    const senderName = event.sender ? event.sender.name : event.getSender();
-    const previousContent = event.getPrevContent() || {};
+    const senderName = event.getSender();
```

Comment:
`getSender` will == `sender.name` if available already. No point doubly checking for it.